src/libtilt/projection/project_real.py

```diff
@@ -75,3 +75,71 @@ def _project_volume(rotation_matrix) -> torch.Tensor:
     xl, xh = padding[2, 0], -padding[2, 1]
     images = [_project_volume(matrix)[yl:yh, xl:xh] for matrix in rotation_matrices]
     return torch.stack(images, dim=0)
+
+
+def project_real_2d(image: torch.Tensor, rotation_matrices: torch.Tensor) -> torch.Tensor:
+    """Make 1D projections of a 2D image in specific orientations.
+
+    Projections are made by
+
+    1. generating a line of coordinates sufficient to cover
+       the image in any orientation.
+
+    2. left-multiplying `rotation matrices` and coordinate grids to
+       produce rotated coordinates.
+
+    3. sampling `image` at rotated coordinates.
+
+    4. summing samples along depth dimension of a `(h, w)` image.
```

Comment:
```suggestion
    4. summing samples along `h` dimension of a `(h, w)` image.
```

you sum over `h` later

Reply:
check